File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/S. Interval/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/S
#include <iostream>
using namespace std;

int main()
{
    float n;
    cin >> n;
    if (n <= 0 || n > 100)
    {
        cout << "Out of Intervals";
    }
    else if (n >= 0 && n <= 25)
    {
        cout << "Interval [0,25]";
    }
    else if (n <= 50)
    {
        cout << "Interval (25,50]";
    }
    else if (n <= 75)
    {
        cout << "Interval (50,75]";
    }
    else
    {
        cout << "Interval (75,100]";
    }
    cout << endl;
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/S. Interval/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/S

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemS
    {
        static void Main(string[] args)
        {
            float n = float.Parse(Console.ReadLine());
            if (n <= 0 || n > 100)
            {
                Console.WriteLine("Out of Intervals");
            }
            else if (n >= 0 && n <= 25)
            {
                Console.WriteLine("Interval [0,25]");
            }
            else if (n <= 50)
            {
                Console.WriteLine("Interval (25,50]");
            }
            else if (n <= 75)
            {
                Console.WriteLine("Interval (50,75]");
            }
            else
            {
                Console.WriteLine("Interval (75,100]");
            }
        }
    }
}
